packages/flutter/lib/src/painting/borders.dart

```diff
@@ -374,13 +410,23 @@ class Border {
     return true;
   }
 
+  Border add(Border typedOther) {
```

Comment:
Docs please

Reply:
Docs are here and will be merged in a subsequent PR, I'm just merging that branch piecemeal:
https://github.com/Hixie/flutter/blob/borders/packages/flutter/lib/src/painting/borders.dart#L279